roles/rust-foundation-project-director.md

```diff
@@ -26,8 +33,25 @@ Please refer to [Rust Foundation Director Roles & Responsibilities][director-rol
 - A keen sense of the needs of Rust Project, its teams, processes, and ecosystem.
 - Willingness to understand and represent viewpoints they may not have personal experience with or necessarily agree with.
 
+## Requirements and Eligibility
+
+There are requirements and eligibility criteria Project Directors must meet.
+These are set by the Rust Project, the [Foundation bylaws], the laws of the State of Delaware, and the laws of the United States of America.
+
+- Project directors must disclose their name on the Foundation's IRS Form 990, which is a public document. For an example, see [the Foundations 2021 filing][form-990].
+- A candidate may not be elected as a Project Director if they are employed by a corporate member of the Foundation that already employs another Project Director.
+  This comes from [Section 4.3(g)] of the Foundation bylaws.
+
+  In the past there has been confusion about the definition of employment.
```

Comment:
This sentence feels kind of out of place.  I would prefer to state that a broad definition has been used in the past and then proceed to describe that definition and not say that there has been confusion.

Reply:
:+1: and I'd also like to lay stress on the legal element here - it isn't a definition we've come up with in isolation. Delaware corporate law generally interprets 'employment' broadly, with the key element being that the individual is engaged to carry out work or services for the company. That can cover a range of different types of relationship.

Reply:
Sounds good. I deleted the line about past confusion and added one about Delaware corporate law.

I do think referencing Delaware law is important here, because it makes it clear that the Foundation isn't just pulling stricter definitions out of thin air.